src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -303,6 +303,7 @@ export const SerchPerformedByExternalComponent: Story = {
           items={filteredItems}
           searchPlaceholder={"Type 'err' for error state"}
           onClick={handleClick}
+          onOpenChange={(isOpen) => isOpen && handleSearch('')}
```

Comment:
Wouldn't it be better to reset it at closing time when there is no risk of seeing strange cases of flickering?